docs/tutorials/geomesa-blobstore-exif.rst

```diff
@@ -0,0 +1,170 @@
+In order to use our new handler, we will need to get some geotagged images.
+Below is a partial Python script that uses the Flickr API to grab some geotagged images taken around Monument Valley.
+Using the Flickr API is beyond the scope of this tutorial, however the Python code snippet is provided for those
+interested in obtaining a test data set. The below script will download 100 images to the user's home directory.
+
+.. code-block:: python
+
+    import flickrapi
+    import requests
+    import itertools
+    api_key = u'Some API Key'
+    secret  = u'some secret'
+
+    flickr = flickrapi.FlickrAPI(api_key, secret)
+    flickr.authenticate_console(perms='read')
+    photos = flickr.photos_search(lat='37.0000', lon='-110.1700', radius='10', safe_search='1', extras='url_o')
+
+    urls = itertools.islice((p.attrib['url_o'] for p in photos[0] if 'url_o' in p.keys()), 100)
+
+    for i, url in enumerate(urls):
+        with open('~/flickr{0:03d}.jpg'.format(i), 'wb') as fd:
+            r = requests.get(url)
+            for chunk in r.iter_content(1024):
+                fd.write(chunk)
+
+
+
+First we need to register a BlobStore using the following command. Note that the parameters for user/password/zookeepers/instanceId need to be modified to suite your environment:
+
+.. code-block:: bash
+
+    $ curl -d 'instanceId=myCloud' -d 'zookeepers=zoo1,zoo2,zoo3' -d 'tableName=myblobstore' -d 'user=user' -d 'password=password' http://localhost:8080/geoserver/geomesa/blobstore/ds/myblobstore
+
+To ingest the files, we can write a simple bash loop to use cURL on each file to ingest to the BlobStore.
+
+.. code-block:: bash
+
+    $ for f in *.jpg; do curl -X POST -F file=@$f http://localhost:8080/geoserver/geomesa/blobstore/blob/myblobstore ; done
+
+The files have now been ingested.
+
+
+Register Index table in GeoServer
+---------------------------------
+
+The BlobStore index DataStore is an ordinary GeoTools DataStore, so the registration in GeoServer is no different than for other DataStores.
+For an example of registering a DataStore in GeoServer please follow these :doc:`/user/geoserver.rst'.
+
+Once the layer is registered we can view the layer in the GeoServer layer previewer.
+
+.. figure:: _static/geomesa-blobstore-exif/blob-index-map.png
+    :alt: Figure of locations of Blobs
+
+
+Querying the Index for Blobs
+----------------------------
+
+To Query for blobs in the spatio-temporal index we will need to enable WFS queries for the layer in GeoServer.
+Ensure the Workspace in which the BlobStore index DataStore was registered has the WFS service enabled.
+This can be verified by clicking through to Workspaces -> (your workspace) -> Checkbox next to WFS -> Save, in the GeoServer UI.
+
+.. figure:: _static/geomesa-blobstore-exif/geoserver-wfs-enable.png
+    :alt: GeoServer Workspace Settings View
+
+This can also be achieved through the GeoServer REST API :doc:`http://docs.geoserver.org/stable/en/user/rest/index.html`
```

Comment:
to do a link, either:

    `GeoServer REST API <http://docs.geoserver.org/stable/en/user/rest/index.html>`__

or if you want to keep the URL in the text, just leave it undecorated (maybe in parentheses) and the Sphinx process will convert it to a hyperlink.